.github/workflows/full-ci.yml

```diff
@@ -117,35 +117,90 @@ jobs:
       (github.event_name == 'pull_request' &&
         github.event.action == 'synchronize' &&
         contains(github.event.pull_request.labels.*.name, 'full-ci'))
-    runs-on: macos-12
+    runs-on: [self-hosted, macOS-12-self-hosted, x86_64, regular]
     steps:
       - uses: actions/checkout@master
         with:
           fetch-depth: 0
           submodules: recursive
 
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+
       - name: Install dependencies
         run: |
-          brew install --overwrite go mage
-          brew install python3
-          pip3 install pytest tarantool requests psutil pyyaml netifaces
-
-      - name: Setup Docker
-        uses: docker-practice/actions-setup-docker@master
+          brew install --overwrite go mage wget
+          pip3 install pytest requests psutil pyyaml netifaces tarantool
 
       - name: Build tt
         env:
           TT_CLI_BUILD_SSL: 'static'
         run: mage build
 
       - name: Install tarantool
-        run: brew install tarantool
+        run: |
+          ./tt init
+          ./tt install tarantool 2.10.7
```

Comment:
Shouldn't we install with `--dynamic` option? It may work faster.

Reply:
changed

Reply:
changed back, unit tests fail with that option enabled